### DISCUSSION TITLE
Question about the addtional wiring re 2nd SID or FM?

### DISCUSSION THREAD

**Discussion by funkychimp** [frntc/SIDKick-pico#89] [General]
Hi,

With regards to using the 2nd SID or FM Expansion. Is this a choice you have to make when adding the extra wires? That is, can you have it wired for both the 2nd SID and FM Expansion at the same time and then choose 'SID+SID' or 'SID+FM' in the config. Or can you only wire for 2 SIDs or SID+FM?

Kind regards

Paul

**Comment by frntc** [comment]
I didn't want to add additional ICs for multiplexing GPIOs and thus only reading two signals was possible on the "classic" variants (the SKpico2040DAC has more GPIOs as solder pads, but not yet used) -- so the answer is "yes and no":

When connecting A6 and IO for FM, you could still select a second SID at $df00. Theoretically, the SID address $d440 would also be possible, I just didn't implement this :-) (it's not very common) But other addresses would require rewiring. I was thinking of making a little PCB which would take all signals and forward those that you want for your configuration.

**Reply by SX64man** [reply]
Ah - that makes sense now. I broke out pins on mb for a5,a6,a7,a8 and i/o1,i/o2 to keep my options open. but could only get a6 and i/o2 combos to work for sid1+FM as designed.   Fantastic work! Thank you for developing these great SiD option.  I sadly have 4 or 5 6581s that have bit the dust in the last year and a few more where the analog filters are disappearing with time. BTW, I made 10 SKPicos the hard way - all components soldered - that took a while... but they all work!  :-)

**Comment by funkychimp** [comment]
Hi,

Thank you for your quick response. I will just wire it up for duel SIDs then. This will get more use as apposed to the limited amount of FM software. Limited at least in comparrison to the number of 2SID tunes available.

Regsrds

Paul

**Comment by sanchomuzax** [comment]
I suggest adding something like this to the `README.md`.

A clear simple summary like this would have helped me a lot. because I got lost between the table and the technical description.

A) If you just want a working second SID . simplest option
**A5/A6 → CPU pin 12 (A5)**
→ gives you a **second SID at $d420**

B) If you want the second SID at $d500
**A8/IO → CPU pin 15 (A8)**

C) If you want both major addresses ($d420 + $d500)
**A5/A6 → CPU pin 12 (A5)**
**A8/IO → CPU pin 15 (A8)**

D) If you also want FM mode (this is the most sensitive setup)
**A5/A6 → CPU pin 13 (A6)**
**A8/IO → Expansion port pin 10 (/IO2)**
This is the **only wiring** where FM is guaranteed to work.
